kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -299,6 +325,19 @@
         message: 'Facility Settings',
         context: 'Title of page where user can configure facility settings.',
       },
+      deviceManagementPin: {
+        message: 'Device management PIN',
+        context: 'The title for the device management pin',
+      },
+      deviceManagementDescription: {
+        message:
+          'This 4-digit PIN allows users to manage content and other settings on learn-only devices',
+        context: 'Description for the device management',
+      },
+      createPinBtn: {
+        message: 'CREATE PIN',
```

Comment:
Non-blocking, but just an "FYI" - the button should handle the capitalization of this for you, automatically 

In other instances when you might need to change the case of the string, you can use various string methods like [toUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase). This can be helpful also if you might need to use the same string with different capitalization in different places.